src/styles/content.scss

```
@@ -0,0 +1,126 @@
+.card {
+  display: grid;
+  justify-content: center;
+  padding: 0 88px;
+
+  &__container {
+    display: grid;
+    row-gap: 46px;
+    column-gap: 48px;
+    grid-template-columns: repeat(4, $card-width);
+
+    padding-top: 50px;
+    padding-bottom: 72px;
+    justify-content: center;
+    margin-left: 48px;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat (2, $card-width);
+    }
+
+    @media (min-width: 768px) {
+      grid-template-columns: repeat (3, $card-width);
+    }
+
+    @media (min-width: 1024px) {
+      grid-template-columns: repeat (4, $card-width);
+      margin: 0 40px;
+    }
+  }
+
+  &__item {
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 0;
+    width: $card-width;
+    transition: transform 300ms;
```

create variable for translateDuration
```suggestion
    transition: transform $translateDuration;
```